packages/react/src/components/_InputWrapper/InputWrapper.tsx

```diff
@@ -201,15 +202,18 @@ const CharacterCounter = ({
       >
         {srLabel}
       </span>
-      <span
-        className={[
-          classes.characterLimitLabel,
-          hasExceededLimit ? classes.characterLimitExceeded : '',
-        ].join(' ')}
-        aria-live={hasExceededLimit ? 'polite' : 'off'}
-      >
-        {label(currentCount)}
-      </span>
+      <div className={classes.characterLimitLabel}>
+        {hasExceededLimit ? (
+          <ErrorMessage
+            as='span'
+            aria-live='polite'
+          >
+            {label(currentCount)}
+          </ErrorMessage>
+        ) : (
+          <Paragraph as='span'>{label(currentCount)}</Paragraph>
+        )}
+      </div>
```

Comment:
Looks a little to big? I see you haven't set the size on the typography components, so they default to `medium`. It does not look like the new Figma design has been updated to use the new tokens, so need input from one of the designers what the size is here. @mrosvik @Febakke 
![image](https://github.com/digdir/designsystem/assets/1070981/f91f3cb0-b729-41a0-8a4d-0a551fc079ce)

![image](https://github.com/digdir/designsystem/assets/1070981/e1ccc83f-6e33-481b-961c-1af0dcae3f1b)

Reply:
Nice catch! Then waiting for @mrosvik or @Febakke before doing anything more. :)

Reply:
Yes, "small" is the best size for this case 👌

![image](https://github.com/digdir/designsystem/assets/1464915/bbd3e1f4-bcea-4c8e-8716-a5b1e787a237)